support-workers/src/main/scala/com/gu/support/workers/exceptions/RetryImplicits.scala

```diff
@@ -23,8 +23,16 @@ object RetryImplicits {
       case e @ (_: SocketTimeoutException | _: SocketException | _: WebServiceHelperError[_]) =>
         new RetryUnlimited(message = e.getMessage, cause = throwable)
 
+      //WebServiceClientError
+      case e @ (clientError: WebServiceClientError) =>
+        if (clientError.codeBody.code == "401"){ // Authentication clientError (401)
```

Comment:
Could you put this as a guard in the case matcher? ` case e @ ... if e.codeBody.code == "401" =>`

Reply:
Also worth expanding briefly on why we're retying 401 now in a comment? Maybe people would find the PR anyway if they were really curious.